sentry/sentry.go

```diff
@@ -7,17 +7,18 @@ import (
 	"github.com/fabric8-services/fabric8-common/auth"
 	"github.com/fabric8-services/fabric8-common/sentry"
 
-	"github.com/getsentry/raven-go"
+	raven "github.com/getsentry/raven-go"
 	"github.com/goadesign/goa/middleware/security/jwt"
 )
 
-type configuration interface {
+// SentryConfiguration the configuration for Sentry
+type Configuration interface {
```

Comment:
why are we exposing this?

Reply:
I guess we could keep it private, but since it's used in the public function, I thought it would make sense to expose it as well. But I can revert that.

Reply:
I'll leave it to you

Reply:
ok, let's keep it as-is for now, then